Makefile

```diff
@@ -210,10 +210,10 @@ delete-remote-tag:
 .PHONY: prepare-tag
 prepare-tag: install-gsed
 	@[ "${TAG}" ] || ( echo ">> env var TAG is not set"; exit 1 )
-	$(SED) -i 's#\(gomod: github.com/SumoLogic/sumologic-otel-collector/.*\) v0.0.0-00010101000000-000000000000#\1 ${TAG}#g' \
+	$(SED) -i'' -e 's#\(gomod: github.com/SumoLogic/sumologic-otel-collector/.*\) v0.0.0-00010101000000-000000000000#\1 ${TAG}#g' \
```

Comment:
Why do we need it if we use `$(SED)`?

Reply:
Good question! Looking at https://github.com/SumoLogic/sumologic-otel-collector/actions/runs/8829744265/job/24241433602, apparently the target depends on `go` being installed. Which is reasonable, but not obvious. I'll modify the workflow instead of making this change.